dynawo/sources/Models/Modelica/Dynawo/Examples/Wind/IEC/IECWT4ANeplan/IECWT4AQControl.mo

```diff
@@ -0,0 +1,100 @@
+within Dynawo.Examples.Wind.IEC.IECWT4ANeplan;
```

Comment:
This case is not working

Reply:
There is a division by 0 in QControl. It seems like a start = U0Pu is not working somewhere. Using a fixed = true in the QControl works but it is not a satisfying solution. I think there is a pb with init.